README.md

```diff
@@ -19,7 +19,21 @@ At the end of the assignment window, simply peruse the candidate's repo and judg
 
 All actions taken by CodeBnb are recorded in the Google Spreadsheet, and you don't have to edit the spreadsheet at all.
 
-## Setup
+## Installation and Setup
+CodeBnb uses [TerraForm](http://terraform.io) to help manage infrastructure.
+
+1. Create an ACM cert valid for your domain, eg `example.com`. The ACM cert should also support a wildcard for your domain, eg `*.example.com`. (Your Lambda function will be called by an API Gateway located at `https://codebnb.example.com`.)
+1. Pick an S3 bucket to use. Set this name in `package.json` under `config` > `s3bucket`.
+1. Create the S3 bucket you want to use to as the source for the AWS Lambda function.
+  1. `$ npm run createS3bucket`
+1. Change to the terraform directory
+  1. `$ cd terraform`
+1. Install the AWS TerraForm provider.
+  1. `$ terraform init`
+1. Deploy everything using TerraForm. `package.json` contains a convenience script which (a) bundles everything into a zip file, (b) uploads it to a version folder in the S3 bucket, (c) sets the s3 bucket and version variables in TerraForm, and (d) runs `terraform apply` for you.
+  1. `$ npm run deploy` and run the `terraform apply` command instructed. You can manually modify the AWS Lambda environment variables afterwards.
```

Comment:
The parent point says that the convenience script does everything, including "(d) runs `terraform apply` for you", but the child bullet says you will be instructed to run `terraform apply` by the script.  One of these is likely wrong!

Reply:
Looking at `package.json`, the parent point is misleading... it should be "(d) shows you the exact `terraform apply` command you need to run." ... but I'm not sure why it _doesn't_ just run `terraform apply` on your behalf, unless there's a good chance you might want/need to alter the command slightly, or perform other interim steps.

Reply:
Fixed up the readme ...  the reason it doesn't run it for you is I encountered issues where terraform failed to properly collect user input interactively from me from variables and complained about missing values. Not 100% sure this wasn't a symptom of another problem though, but for now I'll leave it :)